libevmasm/Assembly.cpp

```diff
@@ -112,7 +112,7 @@ unsigned Assembly::bytesRequired(unsigned subTagSize) const
 namespace
 {
 
-string locationFromSources(StringMap const& _sourceCodes, SourceLocation const& _location)
+string locationFromSources(StringMap _sourceCodes, SourceLocation const& _location)
```

Comment:
This one shouldn't be changed.
If the function only reads from the argument, but does not store/consume it, it's better to keep it at a const reference (otherwise you might force unnecessary copies).
As a rule of thumb: you should only pass by value if you can then ``std::move`` the argument inside the function body.

I saw at least one more such case in ``dev::lll::parseTreeLLL``.